packages/back-end/src/models/OrganizationModel.ts

```diff
@@ -24,6 +24,7 @@ const baseMemberFields = {
       environments: [String],
     },
   ],
+  teams: [],
```

Comment:
Should we type this like we do for `environments` about? E.G `teams: [String]`?

Reply:
Yes, good call